src/odemis/gui/comp/legend.py

```diff
@@ -415,13 +415,16 @@ def on_paint(self, _):
         if self._value_range is None:
             return
 
-        # shared function with the export method
-        self._tick_list, self._vtp_ratio = calculate_ticks(self._value_range, self.ClientSize, self._orientation, self._tick_spacing)
+        offset_y = 0
         csize = self.ClientSize
-
-        if self.lock_va is not None and self.lock_va.value == False:
+        if self.lock_va is not None and self.lock_va.value == True:
             if self._orientation == wx.HORIZONTAL:
                 csize.x -= 24
+            elif self._orientation == wx.VERTICAL:
+                offset_y = 24
+
+        # shared function with the export method
+        self._tick_list, self._vtp_ratio = calculate_ticks(self._value_range, self.ClientSize, self._orientation, self._tick_spacing, start_y=offset_y)
```

Comment:
self.ClientSize -> csize? (I'm not sure, check)

Reply:
Yup. It is.

Reply:
Then pass `csize`?

BTW, I think we should change a little bit the code to read like `csize = self.GetClientSize()`. This is exactly equivalent. However, I find it confusing to read, as setting the .ClientSize actually changes the size of the widget. It's not clear that `csize.x += 24` does *not* change the size of the widget.